config/permissions.yaml

```diff
@@ -13,3 +13,7 @@
     - 100
 "ajax.holidaycredit":
     - 100
+"notifications.index":
+    - 21
+"notifications.save":
+    - 21
```

Comment:
Routes names are wrong in the permissions.yaml.
In the controller, the are called notification.xxxx
So access denied